pkg/runtime/runtime.go

```diff
@@ -1564,16 +1566,15 @@ func (a *DaprRuntime) isAppSubscribedToBinding(binding string) (bool, error) {
 }
 
 func (a *DaprRuntime) initInputBinding(c componentsV1alpha1.Component) error {
-	binding, err := a.bindingsRegistry.CreateInputBinding(c.Spec.Type, c.Spec.Version)
+	fName := fmt.Sprintf(componentFormat, c.ObjectMeta.Name, c.Spec.Type, c.Spec.Version)
```

Comment:
Is it worth refactoring this line into it's own function?

Reply:
Good suggestion. Changed